en/sensor/sfxx_lidar.md

```diff
@@ -0,0 +1,43 @@
+# LightWare SF1XX Lidar
+
+LightWare develops a range of light-weight, general purpose, laser altimeters ("Lidar") suitable for mounting on UAVs. These are useful for applications including terrain following, precision hovering (e.g. for photography), warning of regulatory height limits, anti-collision sensing etc.
+
+![LightWare SF11/C Lidar](../../assets/hardware/sensors/sf11c_120_m.jpg)
+
+PX4 supports the following LightWare Lidar rangefinders through the I2C bus (only):
+* [SF10/A](http://lightware.co.za/shop2017/drone-altimeters/26-sf10a-25-m.html) (25 m)
+* [SF10/B](http://lightware.co.za/shop2017/drone-altimeters/25-sf10b-50-m.html) (50 m)
+* SF10/C (100m) (Discontinued)
+* [SF11/C](http://lightware.co.za/shop2017/drone-altimeters/44-sf11c-120-m.html) (120 m)
+* [LW20](http://lightware.co.za/shop2017/drone-altimeters/51-lw20-100-m.html) (100 m) - Waterproofed (IP67) with servo for sense-and-avoid applications
+
+
+## Configuring Lidar
+
+Newer sensors/versions require no additional configuration.
+
+> **Tip** Older sensors may be miss-configured to have an I2C address equal to `0x55` (this conflicts with `rgbled` module).
+> You can check the address by connecting the sensor via USB (it has an internal USB to serial converter), run terminal, press **space**.
+> If the I2C address is equal to `0x66` the sensor can be used with PX4.
+
+
+## Hardware Setup
+
+Connect the Lidar the autopilot I2C port as shown below (in this case, for the [Pixhawk 1](../flight_controller/mro_pixhawk.md)).
+
+![SF1XX LIDAR to I2C connection](../../assets/hardware/sensors/sf1xx_i2c.jpg)
+
+> **Note** The PX4 driver for this rangefinder only supports I2C connections. You cannot use a UART etc.
```

Comment:
@bkueng @ChristophTobler Is it true that you can only connect to I2C? Looking at the "enabling parameters) it looks like all these are serial [SENS_EN_SF0X](https://docs.px4.io/en/advanced_config/parameter_reference.html#SENS_EN_SF0X) and these must be connected by I2C: [SENS_EN_SF1XX](http://localhost:4000/en/advanced_config/parameter_reference.html#SENS_EN_SF1XX).

If these can be connected by serial, would the connections be as described here: https://pixhawk.org/peripherals/rangefinder#lightware_sf02f_laser_rangefinder_40_m ?

Reply:
@dagar The text here indicates that the SF1XX only supports I2C but as per my comment https://github.com/PX4/px4_user_guide/pull/104/files/c0db40938219d8480ad7f11be04f6853fac349d1#r153391498 it looks like you can use serial. Do you know if this limitation is true or some sort of legacy rubbish? If not, can you suggest someone who might know?

Reply:
Sorry not sure. git blame?

Reply:
Me either. I've asked [on slack](https://px4.slack.com/archives/C3GFT2QS3/p1512534416000033). If no response, maybe create a firmware "question" bug?

Reply:
Oh, are you talking about the PX4 driver by that name or the physical device? Most of them are both i2c and serial. The drivers could probably be merged in PX4 with the interface differences handled internally.

Reply:
I'm talking about the device - the doc is about the device, not the driver. What I would guess has happened is that at some point there was just one driver so this was a true statement. It is also possible that this doc was written just for the I2C driver - which makes little sense. 

Think I can remove the comment safely?

Reply:
Yes I think so.

Reply:
Cheers. Updated in https://github.com/PX4/px4_user_guide/pull/115